rotkehlchen/tests/unit/test_ethereum_airdrops.py

```diff
@@ -28,10 +37,46 @@ def _fixture_airdrop_list(airdrop_list):
 
 @pytest.mark.parametrize('number_of_eth_accounts', [2])
 @pytest.mark.parametrize('use_clean_caching_directory', [True])
-def test_check_airdrops(ethereum_accounts, data_dir):
+def test_check_airdrops(ethereum_accounts, database):
+    # create airdrop claim events to test the claimed attribute
+    tolerance_for_amount_check = FVal('0.1')
```

Comment:
Shouldn't tolerance be smaller? Or wait .. this is just to test the tolerance attribute right?